tests/test_files/long_expressions.py

```diff
@@ -52,3 +52,8 @@ async def bar():
 
 with weird_container[10:str(datetime.datetime.today().date())[:100], None]:
     pass
+
+
+async def async_for_function():
+    async for i in range(10):
+        print(i)
```

Comment:
How about the following, which yields complexity of 9.0 in my testing?

```suggestion
async for i in range(5 * len(str(len(range(2 * 5))) * 2[::-1][0:2] + "foo") / 3):
    pass
```

You'll need to make one more change for this to work - `tests/test_complexity.py` asserts the expected number of errors resulting from a check against `tests/test_files/long_expressions.py`. You will need to alter the assertion on line 9, at the end of `test_fails()` - increment the expected errors by 1, or from the current value of 4 to 5.

Reply:
I've applied the changes I'm talking about in my own fork here: https://github.com/butterflysky/flake8-expression-complexity/tree/derrix060-master

I just don't know how to add those changes to this pull request - I don't have permission to push to your fork @derrix060. Feel free to grab the relevant commits and add them here if you like at your leisure so that hopefully this PR can get merged. I'm looking forward to leaving these checks enabled on my project where I also use `async for`.